awswrangler/s3/_read_parquet.py

```diff
@@ -365,6 +366,7 @@ def read_parquet(
     chunked: Literal[False] = ...,
     use_threads: Union[bool, int] = ...,
     parallelism: int = ...,
+    bulk_read_parquet: bool = ...,
```

Comment:
I'm wondering if we should extract `parallelism` and `bulk_read_parquet` into something like `ray_modin_args`. Both are only applied when we are in distributed mode, and it would make it easier to add more such arguments in the future.

Reply:
Good idea - I am tempted to do that indeed.